server/utils/formUtils.ts

```diff
@@ -0,0 +1,7 @@
+import * as nunjucks from 'nunjucks'
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I think we disabled this globally, it's an annoyance, especially when you know you're going to add stuff later